cogip/tools/detector/__main__.py

```diff
@@ -68,13 +68,6 @@ def main_opt(
         help="Interval between each update of the obstacle list (in seconds)",
         envvar="DETECTOR_REFRESH_INTERVAL",
     ),
-    emulation: bool = typer.Option(
-        False,
-        "-e",
-        "--emulation",
-        help="Force emulation mode.",
-        envvar=["DETECTOR_EMULATION"],
-    ),
```

Comment:
It still can, but since Detector, Server and Planner are now running on the same device, we do not need to send the `virtual` parameter on connection that could be forced by this option.